src/Components/PresentationalComponents/CVETableExpandedCell/CVETableExpandedCell.js

```diff
@@ -8,12 +8,19 @@ import CSAwLabel from '../Snippets/CSAwLabel';
 import CSAwRuleSummary from '../CSAwRuleBox/CSAwRuleSummary';
 import messages from '../../../Messages';
 import { handleCVELink } from '../../../Helpers/VulnerabilityHelper';
+import { NoDescriptionAvailable } from '../EmptyStates/EmptyStates';
 
 const CVETableExpandedCell = ({ description, rules, cve, linkToCustomerPortal }) => {
+
+    const isDescriptionExist = description !== 'No description is available for this CVE.';
+
     return (
         <TextContent className="expanded-cell">
-            <Label className="pf-u-mb-sm"><FormattedMessage {...messages.description} /></Label>
-            <Text component={TextVariants.p}>{description}</Text>
+            {isDescriptionExist && <Label className="pf-u-mb-sm"><FormattedMessage {...messages.description} /></Label> }
+            {
+                isDescriptionExist ?
+                    <Text component={TextVariants.p}>{description}</Text> : <NoDescriptionAvailable/>
+            }
```

Comment:
This two blocks could be merged into one for clarity

`isDescriptionExist ? <Fragment><Label/><Text/></Fragment> : NoDescriptionAvailable`